pkgs/applications/audio/fdkaac/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "fdkaac";
-  version = "1.0.0";
+  version = "v1.0.1";
```

In these cases, just use `1.0.1` as the version:

```suggestion
  version = "1.0.1";
```

And change `rev` below to:

```
    rev = "v${version}";
```